src/utils/proxy-console.js

```diff
@@ -174,7 +174,7 @@
      */
     const styleText = function (textArray, styles = []) {
       return textArray.map((text, index) => {
-        return index ? `<span style="${styles.shift()}">${text}</span>` : text
+        return index ? `<span style="${styles.shift()}">${stringify(text)}</span>` : text
```

Comment:
the item in WIP is referring to this: 

<img width="523" alt="Screen Shot 2019-08-30 at 10 15 25" src="https://user-images.githubusercontent.com/13141462/64028261-4dff4100-cb10-11e9-9971-012ab5700af4.png">

because `styleText` is not expecting arrays at all, but if we stringify whatever arguments that are passed in, the function can handle and return those as strings properly

<img width="636" alt="Screen Shot 2019-08-30 at 10 25 46" src="https://user-images.githubusercontent.com/13141462/64028370-8bfc6500-cb10-11e9-8044-6d67bdd2485b.png">